### ISSUE TITLE
DataTable rendering broken in notebook

### ISSUE THREAD

**Issue by philippjfr** [bokeh/bokeh#6880]
When trying to display a DataTable in the notebook using bokeh 0.12.7 a JS error is generated and the table does not display:

```python
from datetime import date
from random import randint

from bokeh.io import output_notebook, show
from bokeh.layouts import widgetbox
from bokeh.models import ColumnDataSource
from bokeh.models.widgets import DataTable, DateFormatter, TableColumn

output_notebook()

data = dict(
        dates=[date(2014, 3, i+1) for i in range(10)],
        downloads=[randint(0, 100) for i in range(10)],
    )
source = ColumnDataSource(data)

columns = [
        TableColumn(field="dates", title="Date", formatter=DateFormatter()),
        TableColumn(field="downloads", title="Downloads"),
    ]
data_table = DataTable(source=source, columns=columns, width=400, height=280)

show(widgetbox(data_table))
```

```
bokeh-tables-0.12.7.min.js:36 Uncaught Error: SlickGrid's 'enableColumnReorder = true' option requires jquery-ui.sortable module to be loaded
    at init (bokeh-tables-0.12.7.min.js:36)
    at new SlickGrid (bokeh-tables-0.12.7.min.js:38)
    at t.n.DataTableView.t.render (bokeh-tables-0.12.7.min.js:1)
    at t.r.LayoutDOMView.e._layout (bokeh-0.12.7.min.js:29)
    at e.r.LayoutDOMView.e._layout (bokeh-0.12.7.min.js:29)
    at e.r.LayoutDOMView.e._do_layout (bokeh-0.12.7.min.js:29)
    at e.r.LayoutDOMView.e.resize (bokeh-0.12.7.min.js:29)
    at bokeh-0.12.7.min.js:29
```

This seems to be due to the ``reorderable`` option that was added which requires jQuery-UI. When you set ``reorderable=False`` on the ``DataTable`` it works again.

**Comment by mattpap** [comment]
The problem is that data table uses jquery provided by `bokeh-tables.js`, but jquery ui is configured with jquery provided with jupyter, thus things don't work and automatic configuration of `reorderable` fails.

**Comment by svendpeter** [comment]
I have the same problem indeed . Can you kindly tell me,  what is the final solution ? 
Thank you

**Comment by philippjfr** [comment]
It's fixed for me in bokeh versions >= 0.12.9.

**Comment by svendpeter** [comment]
Ok, Thank you
